File: engine/model/src/main/scala/tools/packet/SetNPCScriptable.scala
package tools.packet

import net.opcodes.SendOpcode
import tools.Pair

class SetNPCScriptable(private var _descriptions: java.util.Set[Pair[java.lang.Integer, String]]) extends PacketInput {
  def descriptions: java.util.Set[Pair[java.lang.Integer, String]] = _descriptions

  override def opcode(): SendOpcode = SendOpcode.SET_NPC_SCRIPTABLE
}

File: engine/engine-model/src/main/java/tools/packet/SetNPCScriptable.java
package tools.packet;

import java.util.Set;

import net.opcodes.SendOpcode;
import tools.Pair;

public record SetNPCScriptable(Set<Pair<Integer, String>> descriptions) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.SET_NPC_SCRIPTABLE;
   }
}